src/components/RedactorWrapper/index.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { canUseRedactor } from '../../helpers/featureDetection';
+import RedactorInit from '../../vendor/redactor/redactor';
+import '../../vendor/redactor/_scss/redactor.scss';
+
+export function getEditorContent(editorId) {
+  if (canUseRedactor()) {
+    // use Redactor
+    return window.$R(`#${editorId}`, 'source.getCode');
```

Comment:
this should just be a selector so we don't have a requirement of having an id

Reply:
what do you mean a 'selector'... not sure how you image it